src/fsharp/MethodOverrides.fs

```diff
@@ -153,7 +153,7 @@ module DispatchSlotChecking =
     /// Get the override information for an object expression method being used to implement dispatch slots
     let GetObjectExprOverrideInfo g amap (implty, id: Ident, memberFlags, ty, arityInfo, bindingAttribs, rhsExpr) = 
         // Dissect the type
-        let tps, argInfos, retTy, _ = GetMemberTypeInMemberForm g memberFlags arityInfo ty id.idRange
+        let tps, _, argInfos, retTy, _ = GetMemberTypeInMemberForm g memberFlags arityInfo 0 ty id.idRange
```

Comment:
The "0" reminds me of the "TODO: 0 may be wrong here" comment since it is representing the number of enclosing typars. As far as I know this should be ok? I know I'm being redundant in asking, but I just want to make sure. :)

Reply:
I double checked, this one is OK